docs/cameras.md

```diff
@@ -0,0 +1,183 @@
+---
+comments: true
+---
+
+# Cameras (Terrestrial Observation)
+
+## Definition
+ 
+  Cameras are manual or automatic optical instruments to capture static
+  photos, omnidirectional (360 degrees) photos, virtual reality (VR) photos, or
+  videos. Cameras can be carried by humans (separately or as part of
+  smartphones) or mounted on static spots (e.g., buildings, mountains),
+  drones or satellites [Fact Sheets Drones](drones.md) [and Earth
+  Observation via Satellites](satellites.md).
+ 
+## How to Select a Camera ?
+### Step 1: Are Special Camera Features to be Considered?
+ Most cameras and smartphones allow for storing and extracting **geographical
+  information** along with the photos (highly recommended).
+ 
+  Most allow for **streaming** captured photos/videos automatically to
+  a server in a predefined **frequency** (useful for documenting
+  progress on project sites with mounted cameras).
+ 
+  Some allow for capturing **spherical** photos/videos (which can be
+  used for creating a VR site visit). These photos require compatible
+  players or head-mounted displays to be viewed.
+ 
+  Some [Mobile Data Collection (MDC)](mdc.md) tools work by
+  using the camera of the mobile device to directly link the captured
+  images/videos with other collected information.
+
+### Step 2: What Information Do I Need?
+
+  Cameras can capture, store, and transmit multiple media types that are
+  collected in the field together with the required corresponding
+  meta-data, such as GPS coordinates of the imaging position, recorded
+  direction of the photo (bearing), and the precise time stamp of the
+  imaging (date, hour, minute), which is necessary for the usage of
+  photos for verification purposes. Cameras can be used for various
+  purposes:
+
+-   Baseline documentation (project start)
+
+-   Documentation of project progress on (construction) sites
+
+-   Endline documentation at final inspection and ex-post evaluation
+
+    -   Surveillance of project sites in remote or hard-to-reach
```

Comment:
this bullet points are not in line with the other three, states above. are they sub-points?

Reply:
Indeed they are.